frontend/src/pages/devices/List.tsx

```diff
@@ -78,23 +79,23 @@ export const List = () => {
                 <tbody>
                     {entities.map(entity =>
                         <tr key={ entity.id }>
-                            <th scope="row">
+                            <th className="align-middle">
```

Comment:
I suspect we won't have any screen readers for a while, but I like sticking with the bootstrap default examples. Thoughts on keeping the `scope="row"`? (I don't think it has any visual affects.)

Reply:
It has no visual effects which is why I removed it. I'll put it back in.